### ISSUE TITLE
Builds failing on Jenkins due to Sonar problems: "Can not execute SonarQube analysis: java.io.IOException: Incompatible version 1007"

### ISSUE THREAD

**Issue by karlmdavis** [karlmdavis/rps-tourney#103]
This was on jessentials, but I suspect rps-tourney will fail the same way.

https://justdavis.com/jenkins/job/jessentials/84/console

````
[ERROR] Failed to execute goal org.codehaus.mojo:sonar-maven-plugin:2.2:sonar (default-cli) on project jessentials-aggregator: Can not execute SonarQube analysis: java.io.IOException: Incompatible version 1007. -> [Help 1]
````

**Comment by karlmdavis** [comment]
Yup, failing here, too: https://justdavis.com/jenkins/job/rps-tourney/180/

At a guess, Sonar has to be updated. Which is fine, since I'd wanted to do that anyways.

**Comment by karlmdavis** [comment]
Resolved in commit e278b3a (plus a bunch of sysadmin work the last few days).
